### PR TITLE
Fixed #675 (Picasso)

### LINKED ISSUE #675: Picasso: When clicking and releasing an activity length handle, don't save the unmodified length
ecamp3-unnecessary-save

### PR DESCRIPTION
**Pull request by DeNic0la**
Fixed the issue from
Picasso: When clicking and releasing an activity length handle, don't save the unmodified length #675

### CODE REVIEW

File: frontend/src/components/camp/Picasso.vue
Comment (DeNic0la) [inline frontend/src/components/camp/Picasso.vue]: Since there was no drag (then the if above would be true => no else): if there was no change in length there is no need to update.

**Review by carlobeltrame — CHANGES_REQUESTED** [review]
Nice solution, thanks!
The `_storeData` field is private though, as is denoted in JavaScript by the leading `_`, so I'd prefer if we didn't have to use it since it might change in the future. Maybe you could store the old duration when the dragging is started in `extendBottom`? You can store the value in an additional data property which you can define in `data ()`, similar to e.g. `currentStartTime`.

**Comment by DeNic0la** [comment]
I understand. I think `this.currentEntry.length` was not a good option, since it refers to the object length. There is much code in this File and I understand about half of it, but I think this is a solid Solution.

**Review by carlobeltrame — APPROVED** [review]
LGTM, thanks! Before we can merge, a second member of the ecamp core team needs to approve.

Yeah, this file is way too large, and much of the code is awkward. In part, this is because of the vuetify calendar component which doesn't really allow us to split it further. We're still waiting for vuetify v3 with support for Vue 3, and we hope it will contain a calendar component that is easier to use.

**Review by manuelmeister — APPROVED** [review]
Thank you very much! I can imagine it took a while to understand this code.

**Pull request merged**
